cmd/internal/ecs/add.go

```diff
@@ -520,6 +520,14 @@ func getTaskState(wf *AddWorkflow) (nextState optionals.Optional[AddWorkflowStat
 		return awf_error(errors.Errorf("Error while loading ECS task definition; please contact %s for assistance.", consts.SupportEmail))
 	}
 
+	// Check for bridge networking mode.
+	if output.NetworkMode == types.NetworkModeBridge {
+		printer.Errorf("This task definition is using bridge mode for networking which requires running Insights Agent as a daemon service. " +
+			"However, this is not currently supported by \"ecs add\" command. Please refer to documentation for running Insights Agent as a daemon service, " +
+			"https://learning.postman.com/docs/insights/insights-gs/#configure-the-insights-agent-as-a-daemon-service")
```

Comment:
For some reason, learning center scrolls down to this section when I open the link you added here 
<img width="1920" alt="image" src="https://github.com/postmanlabs/observability-cli/assets/20665864/b05d4918-bf0c-4795-8ac1-ae250ad71a0e">

Reply:
Yes, I will report this to Nermina with a video today.

Reply:
This is likely due to a delay in image load. The browser goes to the point instantly, and the actual point gets shifted when images load.
If you try to reload it repeatedly, the images are fetched from the cache and hence, the page loads correctly.